### PR TITLE
Bugfix/mon 12387

### PR DESCRIPTION
**Pull request by erikdsjostrom**

### CODE REVIEW

**Review by jacobbaungard — APPROVED** [review]
Nice ! Looks good.

Perhaps this is a candidate to go into the 7.5, too?

**Comment by erikdsjostrom** [comment]
> Nice ! Looks good.
> 
> Perhaps this is a candidate to go into the 7.5, too?

Yas

**Comment by kdnilsson** [comment]
Just remember that we have code freeze now, no merge:ing

File: modules/monitoring/includes/downtime.php [hunk 1]
```diff
@@ -148,8 +149,7 @@ public function get_command_mappings($obj_name, $downtime_window, $comment_prefi
 	 */
 	public function get_command($obj_name, $downtime_window, $comment_prefix = 'AUTO: ') {
 		$command = $this->get_command_mappings($obj_name, $downtime_window, $comment_prefix);
-		$cmd_fmt = 'cmd;obj_name;start;end;is_fixed;0;duration;author;comment';
-		return str_replace(array_keys($command), array_values($command), $cmd_fmt);
+		return implode(';', array_values($command));
```
Comment (jimorie) [inline modules/monitoring/includes/downtime.php:152]: I didn't see anything in the official documentation about array_values guaranteeing the order returned, but I guess you convinced yourself of this? :)

Comment (jimorie) [reply modules/monitoring/includes/downtime.php:152]: Nevermind, was too slow. x)

Comment (jacobbaungard) [reply modules/monitoring/includes/downtime.php:152]: The array is generated directly in `get_command_mappings`, as far as I can tell they are always in that order.

**Pull request merged**